BlockchainSdk/Blockchains/TON/TONWalletManager.swift

```diff
@@ -52,9 +53,16 @@ final class TONWalletManager: BaseManager, WalletManager {
     
     func send(_ transaction: Transaction, signer: TransactionSigner) -> AnyPublisher<TransactionSendResult, SendTxError> {
         Just(())
+            .withWeakCaptureOf(self)
+            .flatMap { walletManager, _ in
+                walletManager.getJettonWalletAddressIfNeeded(
+                    for: transaction.sourceAddress,
+                    transactionType: transaction.amount.type
+                )
+            }
```

Comment:
Лучше без Just() тогда, раз можно сразу с подписки начинать 
```suggestion
         getJettonWalletAddressIfNeeded(
               for: transaction.sourceAddress,
               transactionType: transaction.amount.type
          )
```

Reply:
поправил